api/zulip/__init__.py

```diff
@@ -176,6 +176,12 @@ def get_default_config_filename():
                            "  mv ~/.humbugrc ~/.zuliprc\n")
     return config_file
 
+def get_env_variable(keyname):
+    try:
+       return os.environ[keyname]
+    except KeyError:
+       return None
```

Comment:
I think you can just use `os.environ.get(keyname)` in the code below and eliminate this helper variable.